editor/src/uuiui/widgets/popup-list/popup-list.tsx

```diff
@@ -321,7 +323,7 @@ const MenuPortal = (props: MenuPortalProps<SelectOption>) => {
           minWidth: 150,
           maxWidth: 250,
           boxShadow: 'rgba(0, 0, 0, 0.1) 0px 0px 0px 1px, rgba(0, 0, 0, 0.1) 0px 4px 11px',
-          zIndex: 1,
+          zIndex: 999999,
```

Comment:
erm...

Reply:
#ace17cf

Reply:
Did we need to fix this? I think there was a reason I made this change :D